executor/show_test.go

```diff
@@ -1674,3 +1674,31 @@ func (s *testSuite5) TestShowTemporaryTable(c *C) {
 		") ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin AUTO_INCREMENT=2"
 	tk.MustQuery("show create table t7").Check(testkit.Rows("t7 " + expect))
 }
+
+func (s *testSuite5) TestShow(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("use test")
+	tk.MustExec("create table t(id int);")
+	tk.MustExec("create table 中文t(中文id int);")
+	defer tk.Exec("drop table t, 中文t;")
+	// Test for https://github.com/pingcap/tidb/issues/29910
+	tk.MustQuery("show full columns from t like 'ID';").Check(testkit.Rows("id int(11) <nil> YES  <nil>  select,insert,update,references "))
+	tk.MustQuery("show full columns from t like 'id';").Check(testkit.Rows("id int(11) <nil> YES  <nil>  select,insert,update,references "))
+	tk.MustQuery("show full columns from 中文t like '中文ID';").Check(testkit.Rows("中文id int(11) <nil> YES  <nil>  select,insert,update,references "))
+
+	// Test for https://github.com/pingcap/tidb/issues/26253
+	tk.MustQuery("show status like 'ssl_cipher';").Check(testkit.Rows("Ssl_cipher "))
```

Comment:
what if `show status like 'ssl_ciphe%';`

Reply:
if `collation` is not introduced, It is difficult for regexp to ignore case insensitive.